cves/kernel/CVE-2015-8967.yml

```diff
@@ -184,8 +199,9 @@ specification:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  note:
-  answer:
+  note: |
+    There are no notes of a violation of a specification.
```

Comment:
Mabey you could list what sources a couple of the sources you checked

Reply:
Resolved!